### PR TITLE
Adapts to Strapi v4

### PR DESCRIPTION
**Pull request by apriljunge**
(Duplicates #4, but I messed up when pushing updates. So I had to open this one)

Due to an update to Strapi v4 the API has changed.
This commit adapts the current code to work with the new graphql schema.

### CODE REVIEW

**Comment by apriljunge** [comment]
Both of your comments in #4 are fixed now. I had to change the datatype of the variable to work.

**Review by derhuerst — APPROVED** [review]
I'm not familiar with Strapi's response structure, but it seems to work, and the diff doesn't stick out to me.

File: lib/events.js [hunk 1]
```diff
@@ -38,23 +38,26 @@ deepStrictEqual(formatDatetime('2022-03-04T05:06:07+01:00'), [2022, 3, 4, 5, 6])
...
+		? action.attributes.coordinates.split(/,\s+/).map(part => parseFloat(part)).reverse()
 		: null
-	const groupContact = (group?.localizations || [])[0]?.content
+	const groupContact = (group?.data?.attributes.localizations || [])[0]?.data.attributes.content
 	?.find(({__typename}) => __typename === 'ComponentSharedBlocksContact')
 	const organizerUrl = formatOrganizerUrl(
-		group?.city?.slug,
-		group?.city?.federal_country?.slug,
-		group?.city?.federal_country?.country?.slug,
+		group?.data?.attributes.city?.data.attributes.slug,
```
[the remark this answers is not in the document]
Comment (apriljunge) [reply lib/events.js:58]: I assumed that, but I'll change it to be more defensive

**Comment by derhuerst** [comment]
Thanks!

**Comment by derhuerst** [comment]
Damn it, we forgot to update the tests. If you have the time to do this, it would be amazing! Just pass a fixture or two from the API's responses into `parseAction()`.

**Pull request merged**